Adyen/Core/Payment Methods/PaymentMethod.swift

```diff
@@ -75,7 +75,7 @@ public extension PaymentMethod {
     }
     
     /// :nodoc:
-    func localizedDisplayInformation(using parameters: LocalizationParameters?) -> DisplayInformation {
+    func localizedDisplayInformation(using _: LocalizationParameters?) -> DisplayInformation {
```

Comment:
have you check whether this:

```
localizedDisplayInformation(using _: LocalizationParameters?)
```

and this:

```
localizedDisplayInformation(using parameters: LocalizationParameters?)
```

are the same signature?

Reply:
yes, i remember you mentioning this :) will fix